libs/blocks/src/lib/features/base/index.tsx

```diff
@@ -0,0 +1,57 @@
+import clsx from 'clsx';
+import {
+  FluidContainer,
+  Heading,
+  Section,
+  Text,
+  qtJoin,
+} from '@deriv/quill-design';
+import { ReactNode } from 'react';
+
+export interface BaseCardProps {
+  title?: ReactNode;
+  description?: ReactNode;
+  cta?: ReactNode;
+  className?: string;
+  children: ReactNode;
+}
+
+const Base = ({
+  title,
+  description,
+  cta,
+  className,
+  children,
+}: BaseCardProps) => {
+  return (
+    <Section
+      className={clsx(
+        'py-general-4xl',
+        'bg-background-primary-container',
+        className,
+      )}
+    >
+      <FluidContainer className="flex flex-col items-center">
+        {title && (
+          <Heading.H2
+            className={qtJoin(
+              'text-center lg:max-w-[816px]',
+              description && 'pb-general-md',
```

Comment:
Can we add this to the description instead?

Reply:
Because I noticed that the title props is optional so when the title is not passing in it will be an extra padding there
![image](https://github.com/deriv-com/deriv-com-v2/assets/142988136/1909d1c8-9f49-4d87-9a1f-40976cc50003)